### ISSUE TITLE
migration to ListAdapter

### ISSUE THREAD

**Issue by vivchar** [vivchar/RendererRecyclerViewAdapter#32]
Need to investigate this question, looks like will be better if we migrate to ListAdapter.
https://developer.android.com/reference/android/support/v7/recyclerview/extensions/ListAdapter

**Comment by vivchar** [comment]
Currently I rework The Adapter to support [AsyncListDiffer](https://developer.android.com/reference/android/support/v7/recyclerview/extensions/AsyncListDiffer.html)
branch: [feature/async-list-differ](https://github.com/vivchar/RendererRecyclerViewAdapter/tree/feature/async-list-differ)

**Comment by vivchar** [comment]
merged to master, waiting for BETA

**Comment by vivchar** [comment]
```
maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }

dependencies {
    compile 'com.github.vivchar:RendererRecyclerViewAdapter:2.9.0-SNAPSHOT'
}
```

**Comment by vivchar** [comment]
will be released in v3.0
